### ISSUE TITLE
Support to Jasmine 2.0

### ISSUE THREAD

**Issue by emmenko** [mhevery/jasmine-node#282]
http://jasmine.github.io/2.0/introduction.html

Hey guys,

any plan / date in the near future to support the 2.0 version?

Thanks for your work :)

**Comment by aleclarson** [comment]
I forked this repo and updated it to use [**jasmine-core**](https://www.npmjs.com/package/jasmine-core) instead of embedding Jasmine as part of this package. So you should (in theory) be able to use any version of Jasmine you wish. Though it's possible some versions don't work (I've only tested Jasmine 2.2.1).

[**Here's the fork.**](https://github.com/aleclarson/jasmine-node) 

Let me know what you think. Does it suck? Then send me a pull request or make an issue. Thanks!

**Comment by abritinthebay** [comment]
Well Jasmine post 2.0 works totally different so it'll only work for the 2.0 release (and any API compatible version... hopefully they won't change too much in the future).

But that's a nice idea.

**Comment by aleclarson** [comment]
@abritinthebay Yeah I figured. Luckily there's no reason to use my repo if you use any version of Jasmine under 2.0.0 because we have this one! :smile:

**Comment by emmenko** [comment]
PS: this repo has a 2.0 branch, which is also published to npm in case you want to use it.

```
'2.0.0-beta1',
'2.0.0-beta2',
'2.0.0-beta3',
'2.0.0-beta4',
     '2.0.0'
```
